modules/BStorage.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+export class BStorage {
```

Comment:
- [ ] Please don't disable **eslint** because linter checks for those errors which can break your application. :blush: